### PR TITLE
Remove links to ythlsgo.onrender.com

### LINKED ISSUE #14544: Broken: Free to air DD & News 18 channels of India
### Broken Link

https://ythlsgo.onrender.com/channel/UCWk-7Yosyvzln9ZzJg8BvVg.m3u8

### What happened to the stream?

Not loading

### Notes (optional)

Hi!

Many links under country->India have stopped working? **It Seems the problem is with "https://ythlsgo.onrender.com"**
 
**For example the DD channels:**

`DDBharati.in	https://i.imgur.com/4tfUIEo.png	Undefined			DD Bharati (1080p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UCIAnlTV3O9NYWMI-DwYZaWg.m3u8

DDIndia.in	https://i.imgur.com/45uptR8.png	Undefined			DD India (1080p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UCGDQNvybfDDeGTf4GtigXaw.m3u8

DDKashir.in	https://i.imgur.com/1EI4LOf.png	Undefined			DD Kashir (1080p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UCOqU-HfQxoyImaCW2tCDxJA.m3u8

DDKisan.in	https://i.imgur.com/x56WJEa.png	Undefined			DD Kisan (1080p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UCnDfmcUyhgJp6xC1LmBLfUg.m3u8

DDNews.in	https://i.imgur.com/H4f9R7n.png	News			DD News (1080p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UCKwucPzHZ7zCUIf7If-Wo1g.m3u8

DDPunjabi.in	https://i.imgur.com/WVtl6n2.png	Undefined			DD Punjabi (1080p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UC8GYp35wJTN2ituJr0y-7AQ.m3u8

DDSports.in	https://i.imgur.com/J2Ky5OO.png	Sports			DD Sports (576p)	http://103.199.161.254/Content/ddsports/Live/Channel(DDSPORTS)/index.m3u8

DDUrdu.in	https://i.imgur.com/OiQPS34.png	Undefined			DD Urdu (720p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UCu9Fj9QRAg26ZY9DhF8HSrQ.m3u8

DDUttarPradesh.in	https://i.imgur.com/oKsUqBs.png	Undefined			DD Uttar Pradesh (720p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UCxtPipsf2nUZ9xM33TPe93Q.m3u8

DDUttarakhand.in	https://i.imgur.com/GzBsbos.png	Undefined			DD Uttarakhand (480p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UCx55tefMzKkaA0uDmj9g2XA.m3u8
`
**& News 18 channels:**

`News18BiharJharkhand.in	https://i.imgur.com/FQ3zDZE.png	News			News18 Bihar Jharkhand (1080p)	https://ythlsgo.onrender.com/channel/UC531MlZA5LUbeGwEN_zcppw.m3u8

News18India.in	https://i.imgur.com/5VLCwbt.png	News			News18 India (720p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UCPP3etACgdUWvizcES1dJ8Q.m3u8

News18PunjabHaryanaHimachal.in	https://i.imgur.com/iWpFxF7.png	News			News18 Punjab/Haryana/Himachal (720p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UC-crZTQNRzZgzyighTKF0nQ.m3u8

News18Rajasthan.in	https://i.imgur.com/Y7c0omK.png	News			News18 Rajasthan (1080p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UC7FlLbNo66YsCEAPuJITiNg.m3u8

News18Urdu.in	https://i.imgur.com/8xnwMRR.png	News			News18 Urdu (720p) [Not 24/7]	https://ythlsgo.onrender.com/channel/UC9IQhNgS43lmUdU-KdGyb0A.m3u8

News18UttarPradeshUttarakhand.in	https://i.imgur.com/VcwkEDM.png	News			News18 Uttar Pradesh Uttarakhand [Not 24/7]	https://ythlsgo.onrender.com/channel/UCafYgzpyw7aIUYOLjjADu7w.m3u8`

**Plus many more channels.**

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by freearhey**
Closes https://github.com/iptv-org/iptv/issues/14544

Link to download all deleted links: https://freenote.biz/D1qmkPDBmX

### CODE REVIEW

**Comment by Dum4G** [comment]
This removes our knowledge of potentially working sources. Maybe move them to a dedicated playlist that won't be used until we get a stable YT scrapper?

**Comment by freearhey** [comment]
Here's a complete list of all the links removed: https://freenote.biz/D1qmkPDBmX

File: streams/yt [hunk 1]
```diff
@@ -0,0 +1,1411 @@
+#EXTM3U
```
Comment (freearhey) [inline streams/yt:1]: I don't think it's necessary to store links in a repository.

**Comment by freearhey** [comment]
I posted a link to all the links in Discussions as well: https://github.com/orgs/iptv-org/discussions/1535. And as soon as a replacement is found, we will definitely add them back.

**Review by Dum4G — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**